src/main/java/com/google/api/codegen/config/GapicInterfaceConfig.java

```diff
@@ -274,11 +274,7 @@ public GapicMethodConfig getMethodConfig(Method method) {
   }
 
   public GapicMethodConfig getMethodConfig(String methodSimpleName, String fullName) {
```

Comment:
is `fullName` still required as a parameter?

Reply:
Looks like it's not, I'll remove it!